### PR TITLE
chore: update to python 3.7

### PR DESCRIPTION
**Pull request by sbor23**
We used caluma already for some time in a downstream project with python 3.7, so pretty sure we can just increase the version string and should be done with it.

Closes https://github.com/projectcaluma/caluma/issues/773

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.6@sha256:764b2c6beae96a8c8c6046392d5b1b691773337c5a7fd29347935ee0a3223146
+FROM python:3.7.6
```
Comment (ganwell) [inline Dockerfile]: The implications of this have been discussed? I am stickler for hash chains.

Comment (sbor23) [reply Dockerfile]: Not really. My thought process was that maybe renovate bot picks it up this way when the next point release is available, not sure tho.
Would something advice against going `python:3.7.6@sha256:xxx` or is this redundant?

Comment (ganwell) [reply Dockerfile]: Tags can change, digests can not. If you only use the tag the image you are using can change if upstream updates the tag. In 3.6@hash, the tag should become decoration. At least that is what the documentation says.

I guess I could test that with a registry, but I trusted docker in that matter.

Comment (ganwell) [reply Dockerfile]: I tested that.

Comment (sbor23) [reply Dockerfile]: Alrighty did that :)

**Pull request merged**